modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZCardsParagraphBehavior.php

```diff
@@ -149,6 +149,26 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       '#description' => $this->t('Change the color of the Card group title.'),
     ];
 
+    // Card deck border color.
```

Comment:
@mrbeiley I did some investigating and it looks like the behavior plugins have just enough information in their parents array to compute a state selector for the element. Something like this might work:
```suggestion
    // Compute selector for conditional state from array parents.
    $parents = $form['#parents'] ?? [];
    // We're looking to compare to the card style element.
    $parents[] = 'card_style';
    $name = array_shift($parents);
    if (count($parents)) {
      $name .= '[' . implode('][', $parents) . ']';
    }
    $selector = ':input[name="' . $name . '"]';
    // Card deck border color.
```